### PR TITLE
Adding configuration option into the bash and batch script generated by dnu publish

### PR DESCRIPTION
**Pull request by troydai**
Addressing: https://github.com/aspnet/dnx/issues/1501

### CODE REVIEW

**Comment by BrennanConroy** [comment]
:shipit:

**Comment by anurse** [comment]
:shipit:

File: src/Microsoft.Framework.PackageManager/Publish/PublishRoot.cs [hunk 1]
```diff
@@ -106,7 +106,7 @@ private void GenerateBatchFiles()
             }
 
             const string template = @"
-@""{0}{1}.exe"" --appbase ""%~dp0{2}"" Microsoft.Framework.ApplicationHost {3} %*
+@""{0}{1}.exe"" --appbase ""%~dp0{2}"" Microsoft.Framework.ApplicationHost --configuration {3} {4} %*
```
Comment (davidfowl) [inline src/Microsoft.Framework.PackageManager/Publish/PublishRoot.cs]: string interpolation

Comment (troydai) [inline src/Microsoft.Framework.PackageManager/Publish/PublishRoot.cs]: I don't want to update it in this PR so as to limit the scope of the change.

Comment (davidfowl) [inline src/Microsoft.Framework.PackageManager/Publish/PublishRoot.cs]: You totally should though

Comment (troydai) [inline src/Microsoft.Framework.PackageManager/Publish/PublishRoot.cs]: Do you mind if I merge this first then follow up the change to use string interpolation?

**Comment by davidfowl** [comment]
Why is appveyor failing?

**Comment by troydai** [comment]
AppVeyor timeout:
```
Build execution time has reached the maximum allowed time for your plan (40 minutes).
```
According to log, when it timeout the last test was running,

**Pull request merged**